tests/ci/run_formal_verification.sh

```diff
@@ -7,8 +7,13 @@ cd ../
 ROOT=$(pwd)
 
 rm -rf aws-lc-verification-build
-git clone --recurse-submodules https://github.com/awslabs/aws-lc-verification.git aws-lc-verification-build
+git clone https://github.com/awslabs/aws-lc-verification.git aws-lc-verification-build
+
 cd aws-lc-verification-build
+# We avoid pulling the saw-script submodule since the repo will take forever to clone.
+git submodule update —init
```

Comment:
`—init` to `--init`. (Though I'm not sure what is going on with the dashes there)